code-challenges.js

```diff
@@ -47,3 +104,15 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+const multiplier = (number1, number2, number3, number4) => {
+  return number1 - number2 ; number4 - number3
+  // return number4 - number3
+}
+console.log(multiplier(number2, number1))
+console.log(multiplier(number3, number4))
+
```

Comment:
Wow, you got the expected output. It is great to pseudocode when creating functions because there are so many moving parts. Just a note with parameters and arguments. Since the question stated we are taking in two numbers, set the expectation that we will have two parameters in the function and then two arguments in the function call. Then we use methods to get our input to look like our expected output. 
https://github.com/learn-academy-2022-charlie/Syllabus/blob/main/javascript/functions.md